bigquery/spec/datasets_spec.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "spec_helper"
+require_relative "../datasets"
+
+RSpec.describe "List datasets sample" do
+  before do
+    @sample = Samples::BigQuery::Datasets.new
```

Comment:
Lately we've been defining samples using simple top-level methods.  *(this isn't documented as a preferred convention... that's something I will do!)*

See:
 - Datastore [`sample.rb`](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/datastore/sample.rb)
 - Pub/Sub [`sample.rb`](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/pubsub/sample.rb)
 - Logging [`sample.rb`](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/logging/sample.rb)

These are wonderfully simple and no classes/modules are required 😄